testssl.sh

```diff
@@ -20500,6 +20529,7 @@ tuning / connect options (most also can be preset via environment variables):
      --ids-friendly                skips a few vulnerability checks which may cause IDSs to block the scanning IP
      --phone-out                   allow to contact external servers for CRL download and querying OCSP responder
      --add-ca <CA files|CA dir>    path to <CAdir> with *.pem or a comma separated list of CA files to include in trust check
+     --mtls <CLIENT CERT file>     path to <CLIENT CERT> file, it must be in PEM format and contain client certificate with certificate key (not encrypted)
```

Comment:
The line is a bit too long here. It shouldn´t be longer than the one for --color .

Maybe like ``path to <CLIENT CERT> file in PEM format with certificate key, not encrypted``